src/atoms/ComicCard.tsx

```diff
@@ -13,7 +14,7 @@ function ComicCard({ comic }: { comic: IComicData }) {
         />
       </CardBody>
       <CardHeader className="pb-0 pt-2 px-4 flex-col items-start">
```

Comment:
`pb-0` gets overwritten by px-4

Reply:
Thank you, noted. I will deal with this in a another branch (PR)